Update support.rackspace.com.json

kylelaffoon opened the pull request:
Add redirect from Cloud Storage App for Microsoft SharePoint Accessing Cloud Files containers to Cloud Files product page.

kylelaffoon committed:
Update support.rackspace.com.json

Add redirect from Cloud Storage App for Microsoft SharePoint Accessing Cloud Files containers to Cloud Files product page.

kylelaffoon committed:
Update support.rackspace.com.json

kylelaffoon committed:
Update support.rackspace.com.json

smashwilson commented on config/rewrites.d/support.rackspace.com.json:
```diff
@@ -114,5 +114,54 @@
         "rewrite": false,
         "status": 301
       }
+      {
```
You'll need a comma after the `}` :point_up:

smashwilson commented on config/rewrites.d/support.rackspace.com.json:
```diff
@@ -114,5 +114,54 @@
+        "to": "/how-to/cloud-files/",
+        "rewrite": false,
+        "status": 301
+      },
```
You'll also need to remove the final trailing comma here :point_up:

kylelaffoon replied:
Good morning, Ash. Thanks for the heads up on that.

-Kyle

From: Ash Wilson <notifications@github.com<mailto:notifications@github.com>>
Reply-To: rackerlabs/nexus-control <reply@reply.github.com<mailto:reply@reply.github.com>>
Date: Thursday, June 2, 2016 at 7:31 AM
To: rackerlabs/nexus-control <nexus-control@noreply.github.com<mailto:nexus-control@noreply.github.com>>
Cc: Rackspace Hosting <kyle.laffoon@RACKSPACE.COM<mailto:kyle.laffoon@RACKSPACE.COM>>, Author <author@noreply.github.com<mailto:author@noreply.github.com>>
Subject: Re: [rackerlabs/nexus-control] Update support.rackspace.com.json (#479)

In config/rewrites.d/support.rackspace.com.jsonhttps://github.com/rackerlabs/nexus-control/pull/479#discussion_r65530146:

> -        "status": 301
> -      },
> -      {
> -        "description": "Redirect /how-to/cloud-storage-app-for-microsoft-sharepoint-how-to-share-a-file-with-another-person/ URL to /how-to/cloud-files/",
> -        "from": "^\/how-to\/article\/cloud-storage-app-for-microsoft-sharepoint-how-to-share-a-file-with-another-person(.*)$",
> -        "to": "/how-to/cloud-files/",
> -        "rewrite": false,
> -        "status": 301
> -      },
> -      {
> -        "description": "Redirect /how-to/cloud-storage-app-for-microsoft-sharepoint-overview/ URL to /how-to/cloud-files/",
> -        "from": "^\/how-to\/article\/cloud-storage-app-for-microsoft-sharepoint-overview(.*)$",
> -        "to": "/how-to/cloud-files/",
> -        "rewrite": false,
> -        "status": 301
> -      },

You'll also need to remove the final trailing comma here ☝️

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHubhttps://github.com/rackerlabs/nexus-control/pull/479/files/0fcaedf468abaea5e2c27ac677a52cfbfd6e5c71#r65530146, or mute the threadhttps://github.com/notifications/unsubscribe/AFqHDB-_VzmkItK5LeADfmCauORP1weWks5qHs02gaJpZM4Ir_Fu.

kylelaffoon committed:
Remove comma after final statement }.

Replace missing comma in script and remove comma from after the final } within the request.

kylelaffoon commented:
Merge conflict. Starting fresh.
